server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -162,7 +163,7 @@ public void headersShouldBeSetIntoMessage() throws Exception {
                 .bcc(ImmutableList.of(userbcc))
                 .replyTo(ImmutableList.of(userRT))
                 .subject("test subject")
-                .date(ZONED_DATE)
+                .date(Instant.parse("2015-07-14T12:30:42.000Z"))
```

Comment:
I remove ZONED_DATE since it does not make sens anymore. If you read this test fully I think it's clear enough without a constant